### PR TITLE
Flags overwriting config file

### PR DESCRIPTION
**Pull request by mjdesa**
Previously the test file would over write any flags passed to
influx_stress, now flags overwrite config file options

### CODE REVIEW

**Comment by mjdesa** [comment]
@otoolep

File: cmd/influx_stress/influx_stress.go [hunk 1]
```diff
@@ -35,26 +27,46 @@ func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
 	flag.Parse()
 
-	cfg = runner.NewConfig()
+	if *test == "" {
+		fmt.Println("'-test' flag is requeired")
```
Comment (otoolep) [inline cmd/influx_stress/influx_stress.go]: nit: required is misspelled.

File: cmd/influx_stress/influx_stress.go [hunk 2]
```diff
@@ -35,26 +27,46 @@ func main() {
...
+		return
+	}
+
+	cfg, err = runner.DecodeFile(*test)
+
```
Comment (otoolep) [inline cmd/influx_stress/influx_stress.go]: nit: remove this blank line.

**Comment by otoolep** [comment]
+1, thanks.

**Pull request merged**